Database/scr/normalize_numbers.py

```diff
@@ -329,17 +336,23 @@ def _extract_single_number(self, text: str) -> List[float | None] | BaseExceptio
             assert number is not None
         except:
             raise BaseException()
+        if (number == 0 or number == 0.0) and all(
+                    z not in text.lower() for z in self.zero_phrases
+            ):
+                return []
         return [number]
 
     def _extract_numbers_from_tokens(self, doc: spacy.tokens.doc.Doc) -> List[float] | BaseException:
         numbers = []
         tmp_num = ""
         num_ranges = []
+        
         try:
             for i, token in enumerate(doc):
                 if token.tag_ == "CD":
                     try:
-                        num = self._normalize_num(token.text, to_word=True)
+                        #num = self._normalize_num(token.text, to_word=True)
+                        num = self._normalize_num(self.nlp(token.text), to_word=True)
```

Comment:
remove the commented out section
```suggestion
                        num = self._normalize_num(self.nlp(token.text), to_word=True)
```